app/jobs/socure_docv_results_job.rb

```diff
@@ -82,12 +82,13 @@ def perform(document_capture_session_uuid:, async: true, docv_transaction_token_
       )
       record_attempt(
         docv_result_response:,
+        paper_passport: true,
```

Comment:
note: i asked team charity, and when they add passport cards, we will know at this point in the code whether it's a paper passport or a card.

Reply:
Other areas of the IDP use `passport_book` (which might be the same), should we use that instead?